esmvalcore/cmor/_fixes/emac/emac.py

```diff
@@ -67,6 +67,8 @@ def fix_metadata(self, cubes):
         cube = self.get_cube(cubes)
 
         # Fix time, latitude, and longitude coordinates
+        if self.__dict__['extra_facets'].get('fix_time_bounds', False):
+            cube.coord('time').bounds = None
```

Comment:
```suggestion
        if cube.coords("time") and self.extra_facets.get("reset_time_bounds", True):
            cube.coord("time").bounds = None
```

I think it would make sense to make this the default option. As far as I understood the time bounds in EMAC do not have a special meaning.

It would be nice to combine this part with `self.fix_regular_time(cube)` into a `_fix_time(cube)` function that's called here, but that's just a recommendation.

Reply:
Actually I would not make it default True, as it is solved for future production of data and might play a role in future. Although I doubt that it is really breaking anything I would avoid the automatic construction of the time bounds if possible.

Reply:
Fine for me. I just updated the code suggestion above to reflect this.